apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.ts

```diff
@@ -162,6 +180,7 @@ export class DotContentTypesPortletComponent implements OnInit {
     }
 
     private removeIconsFromMenuItem(action: DotActionMenuItem): DotActionMenuItem {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
remove

Reply:
Can't because `icon` is not used because of the destructuring.